README.md

```diff
@@ -177,22 +177,18 @@ Reference sections below in order to know more details about the API
 
 ## API
 
--   FIWARE NGSI v2 [(en)](doc/manuals/user/walkthrough_apiv2.md)
+-   Orion API FIWARE NGSIv2 Walkthrough FIWARE NGSI v2 [(en)](doc/manuals/user/walkthrough_apiv2.md)
     [(jp)](doc/manuals.jp/user/walkthrough_apiv2.md) (Markdown)
--   FIWARE NGSI v2
-    [(en)](http://telefonicaid.github.io/fiware-orion/api/v2/stable/cookbook)
-    [(jp)](https://open-apis.letsfiware.jp/fiware-orion/api/v2/stable/cookbook)
-    (Apiary)
 
 [Top](#top)
 
 ## Reference Documentation
 
 API Reference Documentation:
 
--   FIWARE NGSI v2
-    [(en)](http://telefonicaid.github.io/fiware-orion/api/v2/stable)
-    [(jp)](https://open-apis.letsfiware.jp/fiware-orion/api/v2/stable/) (Apiary)
+-   Orion API FIWARE NGSI v2
+    [(en)](doc/manuals/orion-api.md)
+    [(jp)](doc/manuals.jp/orion-api.md) (Markdown)
```

Comment:
@fisuda jus to check... will this be the path to the orion-api.md Japanese version when PR https://github.com/telefonicaid/fiware-orion/pull/4162 gets merged?

Reply:
Yes. I put orion-api.md in doc/manuals.jp.